cmd/skopeo/proxy.go

```diff
@@ -701,9 +682,13 @@ func (h *proxyHandler) close() {
 		err := image.src.Close()
 		if err != nil {
 			// This shouldn't be fatal
-			logrus.Warnf("Failed to close image %s: %v", transports.ImageName(image.cachedimg.Reference()), err)
+			logrus.Warnf("Failed to close image %s: %v", transports.ImageName(image.img.Reference()), err)
```

Comment:
(Pre-existing, non-blocking: using `image.src.Reference()` here would more obviously correct — but it ends up being the same value in the end.)